cmd/bootnode/main_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 The AthanorLabs/atomic-swap Authors
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/athanorlabs/atomic-swap/cliutil"
+	"github.com/athanorlabs/atomic-swap/common"
+	"github.com/athanorlabs/atomic-swap/daemon"
+	"github.com/athanorlabs/atomic-swap/rpcclient"
+)
+
+func getFreePort(t *testing.T) uint16 {
+	port, err := common.GetFreeTCPPort()
+	require.NoError(t, err)
+	return uint16(port)
+}
+
+func TestBootnode(t *testing.T) {
```

Comment:
I'd like to add unit tests to the `bootnode` package and maybe even use them in the `daemon` package's tests, but this seemed like the most bang-for-the-buck place to introduce initial bootnode unit tests.